### PR TITLE
fix: Change logback default config file to avoid logback.log at the root

### PR DESCRIPTION
**Pull request by nvuillam**
Related to https://github.com/nvuillam/vscode-groovy-lint/issues/200

### CODE REVIEW

**Comment by nvuillam** [comment]
@stevenh I build CodeNarcServer using npm run dev:pre-commit on Windows, but the update check fails anyway, do you have a hint about how to to pass it ?

**Comment by stevenh** [comment]
My guess would be you have an old version of npm which means your zip implementation is creating differently compress output as detailed here: https://github.com/nvuillam/npm-groovy-lint/blob/main/CONTRIBUTING.md#troubleshooting.

For the log file issue I already posted https://github.com/nvuillam/npm-groovy-lint/pull/353

**Comment by stevenh** [comment]
The current version of 18 on github action is v18.19.0

File: lib/java/logback.xml [hunk 1]
```diff
@@ -15,10 +15,16 @@
...
   </appender>
 
-  <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-    <file>${logging.appender.file.fileName:-logback.log}</file>
-    <append>true</append>
-    <immediateFlush>true</immediateFlush>
+  <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <file>${logging.appender.file.fileName:-/tmp/logback-groovy.log}</file>
+    <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+    <fileNamePattern>/tmp/logback-groovy.%d{yyyy-MM-dd}.%i.log</fileNamePattern>
```
Comment (IgorIvanov-Kudan) [inline lib/java/logback.xml:21]: Out of curiosity: wouldn't this break on Windows where there is no `/tmp` directory?

Comment (Artalus) [inline lib/java/logback.xml:21]: Out of curiosity: wouldn't this break on Windows, since there is no `/tmp` directory?

Comment (nvuillam) [reply lib/java/logback.xml:21]: @Artalus i made some tests (I'm on windows) and it was ok, but @stevenh solved it differently on another PR :) ( default log = OFF )

**Pull request closed without merging**